Automattic/wp-calypso
# Guidance: Implement the new mobile master bar

#### Changes proposed in this Pull Request

* In preparation for adding the help center entry point to the masterbar, this PR tucks reader, publish, and profile links in a menu.

This shouldn't change anything in the Desktop experience.

#### Testing instructions

1. In Chrome desktop, go to http://calypso.localhost:3000/
2. Once done loading, open DevTools and simulate mobile.
3. You should see a popover promoting the new navbar.
4. Clicking OK should hide the popover.
5. A menu button should appear and reader, publish, and profile link should be inside it.

I added a don't merge label until we figure persisting the user response and wait for Poli's feedback.

#### Screenshot
<kbd><img width="300" src="https://user-images.githubusercontent.com/17054134/162798985-7974d3ab-f0b6-4022-a9ee-3b76ed456dc8.png"></kbd>
